### PR TITLE
fix(sdk-coin-dot): fix baseFactor for tdot

### PR DESCRIPTION
**Pull request by dpkjnr**
Ticket: WIN-1783

### CODE REVIEW

**Review by gianchandania — APPROVED** [review]

File: modules/sdk-coin-dot/src/dot.ts [hunk 1]
```diff
@@ -361,7 +361,7 @@ export class Dot extends BaseCoin {
     const partialFee = await this.getFee(destAddr, senderAddr, amount);
     // Polkadot has a concept of existential desposit (ed), it is the minimum amount required by an address to have
     // to keep the account active
-    const existentialDeposit = this.getChain() === 'tdot' ? 10000000000 : 1000000000000;
+    const existentialDeposit = 1 * this.getBaseFactor();
```
Comment (ajays97) [inline modules/sdk-coin-dot/src/dot.ts:364]: We can ignore multiplying this by 1 since `this.getBaseFactor()` is enough.

**Review by ajays97 — CHANGES_REQUESTED** [review]

File: modules/sdk-coin-dot/src/dot.ts (hunk 1 above)
Comment (ajays97) [reply modules/sdk-coin-dot/src/dot.ts:364]:
```suggestion
    const existentialDeposit = this.getBaseFactor();
```

**Comment by alebusse** [comment]
Nice catch, im might take over with this pr https://github.com/BitGo/BitGoJS/pull/4219

**Comment by dpkjnr** [comment]
fixed in https://github.com/BitGo/BitGoJS/pull/4219

**Pull request closed without merging**